packages/swr-devtools-panel/package.json

```diff
@@ -36,15 +36,15 @@
     "swr": ">=1.0.0"
   },
   "devDependencies": {
-    "@types/react": "^17.0.16",
+    "@types/react": "^18.0.15",
     "@types/styled-components": "^5.1.25",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
     "styled-components": "^5.3.5",
-    "swr": "^2.0.0-beta.2",
+    "swr": "^2.0.0-beta.6",
     "typescript": "^4.7.4"
   },
   "dependencies": {
-    "react-json-view": "^1.21.3",
+    "react-json-tree": "^0.17.0",
```

Comment:
`react-json-view` hasn't supported React v18 yet, so I've migrated to `react-json-tree`, which is used in Redux DevTools.